linkis-dist/package/sbin/linkis-daemon.sh

```diff
@@ -66,13 +113,18 @@ function start()
       fi
   fi
   export SERVER_START_BIN=$LINKIS_HOME/sbin/ext/linkis-$SERVER_NAME
+  export SERVER_START_COMMON=$LINKIS_HOME/sbin/ext/linkis-common-start
   if [[ ! -f "${SERVER_START_BIN}" ]]; then
       echo "The $SERVER_NAME is wrong or the corresponding startup script does not exist: "
       echo "$SERVER_START_BIN"
       exit 1
   else
       echo "Start server, startup script:  $SERVER_START_BIN"
       export SERVER_NAME=linkis-$SERVER_NAME
+      sed -i "/.*export DEBUG_PORT*/c\#export DEBUG_PORT=" $SERVER_START_BIN
+      sed -i "/.*export DYNAMIC_JVM*/c\#export DYNAMIC_JVM=" $SERVER_START_COMMON
```

Comment:
By modifying the file, there will be persistence problems
Whether to consider adjusting dynamic parameter passing